### ISSUE TITLE
NX Generate UI does not open

### ISSUE THREAD

**Issue by maxbebaasuro** [iguissouma/nx-console-idea-plugin#100]
**Describe the bug:**
NX Generate UI simply does not open and shows no error message.

**Steps to reproduce:**
Right Click -> New -> etc.

**Expected behavior:**
UI Should show up

**Additional context:**
I dont know how to produce debug output

**Comment by maxbebaasuro** [comment]
When running nx generate with --ui from run anything it opens. Only context menu does nothing.

**Comment by maxbebaasuro** [comment]
Is there a chance to get this fixed soonish?

**Comment by maxbebaasuro** [comment]
Hi @iguissouma 

i finally had the change to test what you suggested. And yes, it works in a fresh workspace with preset empty. 
Is there a way to turn on debug output for your plugin? Maybe I can figure our why it doesn't work in the main workspace.

**Comment by maxbebaasuro** [comment]
hm... 
When i reported the nx tasks where still shown. now the list is empty.

**Comment by maxbebaasuro** [comment]
I just had some suspicion. 
We have a strapi project in the same workspace that contains it's own package.json and node_modules. 
As soon as I remove it, the console works

**Comment by maxbebaasuro** [comment]
Ok. This is WEIRD!!!

I deleted all files except of project.json from that strapi project. 
-> Does not work

I deleted the project.json from the strapi project and removed it from workspace.json
-> Does not work

In a fresh workspace with preset empty, i created a folder apps/strapi, copied the project.json in, added it to workspace.json manually
-> Does not work

In a fresh workspace with preset empty, i created a folder apps/strapi, manually created a project.json, copied in the content from the original project.json, added it to workspace.json manually
-> Does not work

In a fresh workspace with preset empty, i created a folder apps/strapi, copied a project.json from ANOTHER project and replaced the project name inside, added it to workspace.json manually
-> Works!!!

This smells so much like some kind of charset issue.
